.eslintrc

```diff
@@ -78,7 +124,6 @@
     "__GOOGLE_ENABLED__": true,
     "socket": true,
     "webpackIsomorphicTools": true,
-    "cy": true,
-    "Cypress": true
+    "cy": true
```

Comment:
@ryansmith94 Is it right that line 127 is still here?

Reply:
I think it actually can go, but I wasn't 100% so I've left it there for now.